src/controllers/single_book.js

```diff
@@ -40,3 +41,18 @@ exports.getSingleStoreBook = (request, response, next) => {
       next(err);
     });
 };
+
+exports.deleteBookFromLibrary = (request, response, next) => {
+  const id = request.body;
+  deleteLibraryBook(id)
+    .then(() => {
+      const result = { message: 'Book is Deleted !' };
+      response.send(JSON.stringify(result));
+    })
+    .catch((err) => {
+      if (err) {
+        response.send(JSON.stringify({ err }));
+      }
+      next(err);
```

Comment:
If your code execute `catch` block means that it already has an `err` so the`response.send(JSON.stringify({ err }))` will send a response, and the `next(err)` will pass the error into `server error` middleware and this middleware will send another response then **It will crash your server**